server/app/services/question/types/CurrencyQuestionDefinition.java

```diff
@@ -28,10 +28,6 @@ public static CurrencyValidationPredicates create() {
     }
   }
 
-  public CurrencyValidationPredicates getCurrencyValidationPredicates() {
```

Comment:
I removed these custom validation predicate methods from questions that don't have custom validation predicates because they are unused. I'm happy to add them back in though if we want to keep them for parity between question types.

Reply:
@jeffpw-goog does this overlap with your work around question validation?

Reply:
Nope - I noticed those unused validation predicates and decided to leave them but I'm indifferent